Resources/public/js/models/ez-locationmodel.js

```diff
@@ -345,6 +345,64 @@ YUI.add('ez-locationmodel', function (Y) {
             return attrs;
         },
 
+        /**
+         * Returns the REST API sort order based on the sortOrder attribute
+         *
+         * @method _getSortDirection
+         * @protected
+         * @return 'ascending' or 'descending'
+         */
+        _getSortDirection: function () {
+            return this.get('sortOrder') === 'ASC' ? 'ascending' : 'descending';
+        },
+
+        /**
+         * Returns the REST API sort clause identifier based on the sortField
+         * attribute.
+         *
+         * @method _getSortClauseIdentifier
+         * @protected
+         * @return {String}
+         */
+        _getSortClauseIdentifier: function () {
+            switch(this.get('sortField')) {
+                case 'MODIFIED':
+                    return 'DateModified';
+                case 'PUBLISHED':
+                    return 'DatePublished';
+                case 'PATH':
+                    return 'LocationPath'; // Not Implemented server side !?!
+                case 'SECTION':
+                    return 'SectionIdentifier';
+                case 'DEPTH':
+                    return 'LocationDepth';
+                case 'CLASS_IDENTIFIER':
+                    return 'ClassIdentifier'; // Not implemented server side !?!
+                case 'CLASS_NAME':
+                    return 'ClassName'; // Not Implemented server side !?!
```

Comment:
These two will not be implemented.

Could for instance _(since warning is not needed in this case, swap out `console.log` if something else if better)_:
```diff
+                case 'CLASS_IDENTIFIER':
+                case 'CLASS_NAME':
+                    console.log("Deprecated sort field used '" + this.get('sortField') + "', falling back to 'DateModified'.");
+                    return 'DateModified';
```